aws/fedora-coreos/kubernetes/workers/fcc/worker.yaml

```diff
@@ -3,7 +3,7 @@ variant: fcos
 version: 1.1.0
 systemd:
   units:
-    - name: docker.service
+    - name: containerd.service
       enabled: true
```

Comment:
For containerd add an override drop-in:
```
ExecStartPre=-/bin/sh -c "for f in /usr/libexec/cni/*; do ln -s \"$f\" /opt/cni/bin/$(basename $f); done"
```
This symlinks in the cni plugins that are installed in `/usr/libexec/cni`

Reply:
CNI plugins are typically added by daemonset sidecars